### PR TITLE
Documentation for OpenTherm Gateway climate platform

### PR DESCRIPTION
**Pull request by mvn23**
**Description:**
Add documentation for the OpenTherm Gateway climate platform.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#16299
## Checklist:

- [X] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/climate.opentherm_gw.markdown [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+The `opentherm_gw` climate platform is used to control the [OpenTherm Gateway](http://otgw.tclcode.com/) from Home Assistant.
+
+# {% linkable_title Configuration %}
+
+```yaml
+# Example configuration.yaml entry
+climate:
+  - platform: opentherm_gw
+    device: /dev/ttyUSB0
+    name: Thermostat
```
Comment (MartinHjelmare) [inline source/_components/climate.opentherm_gw.markdown]: Only include the required options to make the first example minimal.

**Review by fabaff — DISMISSED** [review]
Thanks :bird:

**Comment by mvn23** [comment]
Thanks! I was not aware of the 'default' option for the configuration parameters.
I am considering to add a note about the update speed of the thermostat. As the OpenTherm protocol is dependant on polling from the thermostat, it can take some time for changes to be propagated between Home Assistant, the boiler and the thermostat. Would this page be the right place to add a note like that or would this be considered a limitation of the library/backend and thus have no place in Home Assistant's docs?

**Comment by fabaff** [comment]
Everything that could be helpful for users should have a place in the docs.

After the PR is merged can changes be added directly through the GitHub editor.

**Review by fabaff — APPROVED** [review]

**Pull request merged**